matsim-jerusalem/src/jerusalem/scenario/RunJerusalem.java

```diff
@@ -0,0 +1,234 @@
+package jerusalem.scenario;
+
+import java.util.Arrays;
+import java.util.EnumSet;
+
+import org.matsim.api.core.v01.Scenario;
+import org.matsim.api.core.v01.TransportMode;
+import org.matsim.core.config.Config;
+import org.matsim.core.config.ConfigUtils;
+import org.matsim.core.config.groups.ControlerConfigGroup.EventsFileFormat;
+import org.matsim.core.config.groups.ControlerConfigGroup.RoutingAlgorithmType;
+import org.matsim.core.config.groups.PlanCalcScoreConfigGroup;
+import org.matsim.core.config.groups.PlanCalcScoreConfigGroup.ActivityParams;
+import org.matsim.core.config.groups.PlanCalcScoreConfigGroup.ModeParams;
+import org.matsim.core.config.groups.PlansCalcRouteConfigGroup.ModeRoutingParams;
+import org.matsim.core.config.groups.QSimConfigGroup.TrafficDynamics;
+import org.matsim.core.config.groups.StrategyConfigGroup.StrategySettings;
+import org.matsim.core.controler.Controler;
+import org.matsim.core.controler.OutputDirectoryHierarchy.OverwriteFileSetting;
+import org.matsim.core.replanning.strategies.DefaultPlanStrategiesModule;
+import org.matsim.core.scenario.ScenarioUtils;
+import org.matsim.core.scoring.functions.ScoringParameters;
+
+/**
+ * @author Golan Ben-Dor
+ */
+public class RunJerusalem
+{
+	final public static String INPUT_NETWORK = "E:\\Golan\\Jerusalem MATSim\\MATSim Input\\network\\4.network_2015_no_connectors_multi_modal_cleaned.xml";
+	final public static String INPUT_POPULATION = "E:\\Golan\\Jerusalem MATSim\\MATSim Input\\population\\2.population_centriods_ido_taxi_as_cars.xml";
+	final public static String OUTPUT_FOLDER = "E:\\Golan\\Jerusalem MATSim\\MATSim output\\";
+	final public static String RUN_ID = "" + 5;
```

Comment:
why does rune_id equals to 5?

Reply:
Because I tested different runs to see if MATSim crashes in different parts of the simulation (larger iteration numbers).
I have an excel table where I specify what's inside each run output